doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,605 @@
+add the internal IP address of all border routers of the local AS to the DRKey delegation list for FABRID.
+
+This could look like this::
+
+    [drkey.level1_db]
+    connection = "gen-cache/cs1-ff00_0_110-1.drkey-level1.db"
+
+    [drkey.secret_value_db]
+    connection = "gen-cache/cs1-ff00_0_110-1.drkey-secret.db"
+
+    [drkey.delegation]
+    FABRID = [ "fd00:f00d:cafe::7f00:11", "fd00:f00d:cafe::7f00:12", "fd00:f00d:cafe::7f00:13"]
+
+
+Configuring FABRID Policies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+FABRID policies are configured in the control service using YAML files. A YAML configuration contains the information necessary to create entries in the SupportedIndicesMap, IndexIdentifierMap, IdentifierDescriptionMap (for local policies) and the MPLSMaps. Concretely the YAML file should contain the following entries:
+
+* ``local`` (bool): Indicating whether the policy is a local policy (true) or global policy (false).
+* ``local_description`` (string): The description that is fetched by remote AS'es for this specific policy. Only used for local policies, as for global policies this is stored in a global datastore. Required when ``local`` is true, ignored otherwise.
+* ``local_identifier`` (integer): The integer identifier that this policy is known by. Required when ``local`` is true, ignored otherwise.
+* ``global_identifier`` (integer): The integer identifier of the policy from the global datastore that this configured policy should implement. Required when ``local`` is false, ignored otherwise.
+* ``connections`` (list of ConnectionPoints): The connection points on which this policy applies.
+
+
+Connection Points
+'''''''''''''''''''
+A connection point in the YAML file is defined by providing the ingress and egress of the connection as well as the integer MPLS label that should be applied to enforce the policy on that connection. The egress can be either an interface, an IPv4/IPv6 prefix or a wildcard. The ingress of a connection point is limited to a wildcard or interface.The rationale behind this is that an IPv4/IPv6 ingress would indicate a packet coming from an endhost. Upon arrival at the border router, the packet would have already traversed the internal network and the border router would not be able to enforce a policy (e.g. by applying an MPLS label).
+
+Example of a list of connection points:
+
+::
+
+    - ingress:
+          type: interface
+          interface: 2
+      egress:
+          type: interface
+          interface: 1
+      mpls_label: 1    
+    - ingress:
+          type: interface
+          interface: 2
+      egress:
+          type: ipv4
+          ip: 192.168.5.1
+          prefix: 24
+      mpls_label: 55
+    - ingress:
+          type: wildcard
+      egress:
+          type: interface
+          interface: 2
+      mpls_label: 3
+
+
+
+Border router
+^^^^^^^^^^^^^^^
+
+For a router to query the DRKey secret value from the control service, once has to enable this.
+
+This could look like this::
```

Comment:
Instead of "This could look like this" please directly use the name of the file that needs to be modified -> "Example (X.txt):"

And similarly for all other file contents presented in this section.

Reply:
I changed this for cs1-ff00_0_110-1.toml and br1-ff00_0_110-1.toml but I don't know whether the yaml files need to have any specific constraints on their name or whether they could just be called e.g. "policy1.yaml". @jeltevanbommel

Reply:
They can be arbitrarily named within their folder. I've added a sentence on this folder to configuration of control service for completeness.

Reply:
Done